app/utilities/schema.py

```diff
@@ -25,6 +27,41 @@
 }
 
 
+def get_schema_path_map_for_language(language_code):
+    schema_dirs = [
+        os.path.join(schema_dir, language_code) for schema_dir in SCHEMA_DIRS
+    ]
+
+    schema_files = []
+
+    for schema_dir in schema_dirs:
+        schema_files.extend(glob(f"{schema_dir}/*.json"))
+
+    return {
+        Path(schema_file).with_suffix("").name: schema_file
+        for schema_file in schema_files
+    }
+
+
+def get_schema_path_map():
+    language_map_codes = ["en", "cy", "ga", "eo"]
+
+    return {
+        language_code: get_schema_path_map_for_language(language_code)
+        for language_code in language_map_codes
+    }
+
+
+SCHEMA_PATH_MAP = get_schema_path_map()
+
+
+def schema_exists(language_code, schema_name):
+    return (
+        language_code in SCHEMA_PATH_MAP
+        and schema_name in SCHEMA_PATH_MAP[language_code]
```

Comment:
If `language_code` isn't a key in `SCHEMA_PATH_MAP` this would fail with a KeyError, so the first `in` operation is redundant

Reply:
The first in operation is to prevent the key error and make this method that returns a boolean?

Reply:
Ah yeah, my mistake it will short circuit :+1: